src/main/java/net/mcreator/workspace/misc/WorkspaceInfo.java

```diff
@@ -204,4 +205,8 @@ public Workspace getWorkspace() {
 		return workspace;
 	}
 
+	public MItemBlock itemBlock(String itemBlock) {

```

Comment:
Maybe you should keep `getWorkspace()` the last method (move the new one before it)?